.github/workflows/consistency-test.yml

```diff
@@ -0,0 +1,91 @@
+name: consistency_testing
+on:
+  repository_dispatch:
+    types: [chaos-test-command]
+
+jobs:
+  run_consistency_tests:
+    runs-on: ubuntu-20.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      CI: 1
+      CI_INSTALL_INFRA: 1
+      AWS_EC2_METADATA_DISABLED: true
+
+    steps:
+      - name: Setup gcloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          project_id: 'redpandaci'
+          service_account_key: ${{ secrets.GCR_JSON}}
+          export_default_credentials: true
+
+      - name: setup env slash command
+        run: |
+          echo "Populating env from PR context"
+          commit=${{ github.event.client_payload.pull_request.head.sha }}
+          short=${commit:0:7}
+
+          # Make env available to next steps
+          echo "deb_pkg_url=gs://vectorizedio/rp_dev/release/clang/$short/redpanda_*.deb" >> $GITHUB_ENV
+          echo "upload_dir=redpanda/pr-${{ github.event.client_payload.pull_request.number }}" >> $GITHUB_ENV
+        if: ${{ github.event.action == 'chaos-test-command' }}
+
+      - name: fetch release from gcb
+        run: |
+          # check if the file exists
+          check=$(gsutil -q stat $deb_pkg_url || echo "NotFound")
+          if [[ $check == "NotFound" ]]; then
+            echo "Provided file $deb_pkg_url was not found"; exit 1
+          fi
+
+          gsutil cp $deb_pkg_url redpanda.deb
+
+      - name: Setup vtools
+        run: |
+          sudo apt-get install python3-pip unzip python3-setuptools python3-venv
+          python3 -m venv .chaosenv && source .chaosenv/bin/activate
+          pip install git+https://${{ secrets.GH_API_TOKEN }}@github.com/vectorizedio/vtools > vtools_out.log 2>&1
```

Comment:
This should probably use the script that will install a given vtools version, after that PR's merged, of course.

Reply:
Since the other PR is still under discussion. Let's add that as a next step.